lib/editor/components/timetable/TimetableEditor.js

```diff
@@ -219,8 +219,12 @@ export default class TimetableEditor extends Component<Props, State> {
         // }
         // if (stop.timepoint === null || stop.timepoint) {
         objectPath.set(newRow, `stopTimes.${i}.departureTime`, patternHaltIsStop(halt) ? cumulativeTravelTime : null)
+        objectPath.set(newRow, `stopTimes.${i}.pickupType`, halt.pickupType)
+        objectPath.set(newRow, `stopTimes.${i}.dropOffType`, halt.dropOffType)
         objectPath.set(newRow, `stopTimes.${i}.continuousPickup`, halt.continuousPickup)
         objectPath.set(newRow, `stopTimes.${i}.continuousDropOff`, halt.continuousDropOff)
+        objectPath.set(newRow, `stopTimes.${i}.pickupBookingRuleId`, halt.pickupBookingRuleId)
+        objectPath.set(newRow, `stopTimes.${i}.dropOffBookingRuleId`, halt.dropOffBookingRuleId)
```

Comment:
I'm now seeing the same issue with "stop_headsign". Assuming it needs added here as well?

Reply:
@philip-cline Now that dev-flex has been merged into this branch, should the stop_headsign parameter be present here now? I'm not seeing it so concerned this is still going to fail.

Reply:
@binh-dam-ibigroup Afraid not. https://github.com/ibi-group/datatools-ui/pull/910 doesn't contain an update to the TimetableEditor. I would expect to see something like the following:

`objectPath.set(newRow, `stopTimes.${i}.stop_headsign`, halt.stop_headsign)`

Reply:
@br648 Then it looks like you are referring to #890, which looking at the commit history, is not in the `dev-flex` branch.

Reply:
@philip-cline and I will merge `dev` into `dev-flex` soon -- there are some merge conflicts we need to block out time to resolve